src/api/lib/tasks/dev/notifications.rake

```diff
@@ -30,6 +30,8 @@ namespace :dev do
       end
 
       repetitions.times do |repetition|
+        puts 'Creating entries for which a notification is automatically generated...'
```

Comment:
I would just remove all those puts, it's not really needed, as we can do nothing with that information. I would just put a line indicating how many records are being created at the start of the process and that's all

Reply:
I dropped the redundant text puts, it was pretty useless, I agree. :+1: